website/src/js/PlaygroundViewModel.ts

```diff
@@ -306,6 +306,8 @@ export class PlaygroundViewModel {
 		}
 
 		this.code('<?php declare(strict_types = 1);\n' +
+			'use function PHPStan\\dumpType;\n' +
```

Comment:
Please one more newline after the declare. I don't like the use statements to stick too close to the first line.

Reply:
done